### ISSUE TITLE
Servo attempts to parse badcert.html as CSS rules

### ISSUE THREAD

**Issue by dagnir** [servo/servo#10095]
When running `RUST_LOG="debug" ./mach -d -- https://ideone.com` I see the following error:

```
INFO:script::reporter: 1:1 Invalid rule: '<html>
<head>
<title>Certificate error</title>
</head>
<body>
    <img src="badcert.jpg">
</body>
</html>
'
```

The message seems to be generated from [components/style/stylesheets.rs#138](https://github.com/servo/servo/blob/master/components/style/stylesheets.rs#L138).

**Comment by jdm** [comment]
Ah, this occurs when there's a stylesheet link that yields an error. There's an open PR to address this.

**Comment by mbrubeck** [comment]
Is this fixed? I can no longer reproduce it at ideone.com but I don't know whether the site has changed.

**Comment by jdm** [comment]
This was fixed by #9942.
